mne_bids/meg_bids.py

```diff
@@ -362,6 +372,14 @@ def raw_to_bids(subject_id, task, raw_file, output_path, session_id=None,
             else:
                 raise ValueError('"%s" already exists. Please set overwrite to'
                                  ' True.' % raw_file_bids)
+        """
```

Comment:
this seems to be commented out?

Reply:
Oh yeah, I commented it out because this code causes the check to fail because it doesn't expect there to be raw files in that location. I think I raised it somewhere, but it is because the specification is still undecided on whether the raw files should be in a sub-folder or not (see my comment on the BIDS specification in section 8.4.1 about this...)
I guess I should have just removed it